release/release_test.go

```diff
@@ -109,8 +109,8 @@ func (s *ReleaseTestSuite) TestForceDevMode(c *C) {
 		// Please keep this list sorted
 		{id: "arch", devmode: true},
 		{id: "debian", devmode: true},
-		{id: "elementary", devmode: true},
-		{id: "elementary", idVersion: "0.4", devmode: false},
+		{id: "elementary OS", devmode: true},
+		{id: "elementary OS", idVersion: "0.4", devmode: false},
```

Comment:
This should never have spaces: _A lower-case string (no spaces or other characters outside of 0–9, a–z, ".", "_" and "-")_